### ISSUE TITLE
No image with 600TVL mini-cam

### ISSUE THREAD

**Issue by ABLomas** [PlayUAV/PlayuavOSD#19]
One of most popular cameras for FPV is PZ0420 and it's copies (for example, mini-version: http://www.surveilzone.com/DC5V-plastic-case-mini-sony-super-had-ccd-600tvl-super-camera-2.8mm-lens-for-fpv )
It uses Nextchip 2040 DSP, is cheap enough and produces acceptable image at all lightning conditions.
But i can't make it work with PlayuavOSD. I had full size old PZ0420, several later clones, mini version and so on - with all of them i get only camera image OR (sometimes) only OSD.
Connected Effio-A camera - everything works OK, i got image with overlay.
Re-connected old camera - and again, no image.

Video level may be off on some cameras (i will connect to scope next week), but not on all for sure. There's something more...
On MW-OSD some users need to uncomment VSYNC and FASTPIXEL to get them working, for some it's reverse (it does NOT work with VSYNC). Maybe we have this issue because VSYNC is forced in PlayuavOSD?

**Comment by ghost** [comment]
Hi,

I got a SONY 700TVL camera. It does have the issue as your description. I will take a look the solution of MW_OSD and try to find if it can be solved on PlayUAV OSD board. Thanks for your reporting.

**Comment by ABLomas** [comment]
Looks like we need to swap chip: http://en.playuav.com/question/21
This is not big issue, can do that - only problem is where to get that AT7456 chip - nobody knows about it.
Only sources i can find - taobao and similar CN sites, without english frontend. Not even datasheet (aliexpress sells DP7456 which is NOT the same?)...
As i have 7x PlayUAV OSD's, need maybe 10 of them, anyone know reliable source?

**Comment by TobiasBales** [comment]
@ABLomas would you like to figure this out and try to get it fixed?
I picked up development (on my fork) and would like to get those things out of the way.
If so just shoot me a mail at myname@gmail.com

**Comment by ABLomas** [comment]
Well, i still have lots of those OSD's around, so...
Btw, chip is not real issue - AT7456 and old MAX chip - both do not work properly with Nextchip 2040 DSP.

**Comment by ABLomas** [comment]
Oh, and thanks for doing something with this OSD ;-)
I see you did many commits to configurator, i check this out soon. Real issue is home arrow (https://github.com/PlayUAV/PlayuavOSD/issues/29) - while sync issues can be fixed on ground, improper distance/direction calculations could easily lead to loss of aircraft...

**Comment by TobiasBales** [comment]
Yeah I looked over the ticket yesterday, saw it but didn't take a look yet.
Will check out the arrow today and see about the distance issues.
I thought there was an issue with the imperial units but maybe there is something wrong with the metric ones too, will report back.

Btw: I activated issues on my fork since the maintainers of this org do not respond, even on the playuav.com website.
